buildenv/jenkins/testJobTemplate

```diff
@@ -246,6 +245,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('UPSTREAM_TEST_JOB_NAME', "", "Auto-populated. Upstream test job name. It will be used together with PARALLEL=Dynamic")
 							stringParam('UPSTREAM_TEST_JOB_NUMBER', "", "Auto-populated. Upstream test job number. It will be used together with PARALLEL=Dynamic")
 							booleanParam('LIGHT_WEIGHT_CHECKOUT', LIGHT_WEIGHT_CHECKOUT.toBoolean(), "Optional. Default is false for Grinders")
+							stringParam('CLOUD_PROVIDER', CLOUD_PROVIDER, "available cloud provider")
```

Comment:
Minor nitpick, capitalize **A**vailable to match the other param descriptors that are capitalized

Reply:
Since we are updating this line, could you also provide examples?

Reply:
Updated.